tools/pubsys/src/aws/publish_ami/mod.rs

```diff
@@ -348,9 +345,9 @@ pub(crate) async fn modify_snapshots(
 pub(crate) async fn modify_regional_snapshots(
     user_ids: Option<Vec<String>>,
     group_names: Option<Vec<String>>,
-    operation: &str,
-    snapshots: &HashMap<Region, Vec<String>>,
-    clients: &HashMap<Region, Ec2Client>,
+    operation: OperationType,
```

Comment:
Can this just take a reference instead, so that callers don't have to clone it?
```suggestion
    operation: &OperationType,
```

(Same note for the other functions that now take an `OperationType`.)